x-pack/legacy/plugins/ml/server/models/data_visualizer/data_visualizer.ts

```diff
@@ -4,7 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { CallAPIOptions, RequestHandlerContext } from 'kibana/server';
+import { CallAPIOptions } from 'kibana/server';
+import { IScopedClusterClient } from 'src/core/server';
```

Comment:
NIT: `kibana/server` and `src/core/server` are the same alias. Use only one import line.

Reply:
updated in f0bac33e423d5a570bfa2ba8157a9d1e677c6afb